libr/main/rabin2.c

```diff
@@ -704,8 +747,10 @@ R_API int r_main_rabin2(int argc, char **argv) {
 			if (is_active (R_BIN_REQ_SECTIONS)) {
 				action &= ~R_BIN_REQ_SECTIONS;
 				action |= R_BIN_REQ_SEGMENTS;
+				sss += 1;
 			} else {
 				set_action (R_BIN_REQ_SECTIONS);
+				sss += 1;
```

Comment:
same operation in if and else. just put that statement below outside the conditional blocks